inc/tflite_msg.hpp

```diff
@@ -0,0 +1,49 @@
+//  For licensing see accompanying LICENSE file.
+//  Copyright © 2024 Argmax, Inc. All rights reserved.
+#pragma once
+
+#include <string>
+#include <memory>
+#include <thread>
+#include <condition_variable>
+#include <mutex>
+
+#ifdef ANDROID_JNI
+#include <android/log.h>
+
+#define TAG "com.argmax.whisperax"
```

Comment:
TAG is unsafe name for a preprocessor macro  

ARGMAX_WHISPERKIT_BUNDLE_INFO

can be constexpr const char* 

We can also put these logging methods in to a namespace / utils struct